README.md

```diff
@@ -168,7 +168,7 @@ The non-hook APIs just expose the `useResponsive` hook with different APIs, for
 
 ## Contributing
 
-Read the [Contributing guidelines](../docs/CONTRIBUTING.md)
+Read the [Contributing guidelines](https://github.com/Farfetch/react-context-responsive/blob/master/docs/CONTRIBUTING.md)
```

Comment:
It shouldn't be an absolute path

Reply:
You're right 👍

I've changed the [Contributing guidelines] link path to (./docs/CONTRIBUTING.md) and I've also changed [MIT] link path to (./LICENSE)

(Sorry for all the commits 😅  )

Reply:
Maybe you could squash every commit